app/views/step_by_step_pages/show/_required_actions.html.erb

```diff
@@ -38,6 +38,12 @@
           </li>
         <% end %>
       <% end %>
+
+      <% if %w(draft submitted_for_2i).include?(@step_by_step_page.status) %>
```

Comment:
I wonder if this should be split in to different actions? 
e.g. if status is `draft`, the action should be "Submit for 2i" 
if status is `submitted for 2i` and user is not the drafter, the action should be "2i changes"
if status is `submitted for 2i` and user is the drafter, the action should be "2i review required"
if status is `in_review`, the action is ....

Ideally if there was more time, these could be encapsulated in methods, one for each action, e.g. `fix_broken_links?`.

Disclaimer: the words need more thought.